MetaMask/metamask-extension
# handle 'onTouchStart' and 'onTouchEnd' events in HoldToRevealButton

to make SRP(Secret Recovery Phrase) can be revealed in Mobile Browser

## Explanation

HoldToRevealButton rely on  'OnMouseDown' and 'OnMouseUp' events to track down a bool state called 'isLongPressing', which will not work on Mobile Devices. When running in mobile devices, react-js fires  'onTouchStart' and 'onTouchEnd' events instead when user long press any button.


* Fixes #19254

## Manual Testing Steps

1 install metamask extension in Mobile Browser supports extensions(Mises Browser )
2 click 'Settings' then 'Reveal Secret Recovery Phrase''
3 hold the 'Hold to reveal SRP' button , now it should works, and the SRP reveals.


## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
